src/shared/cloudformation/templateRegistry.ts

```diff
@@ -43,6 +50,7 @@ export class CloudFormationTemplateRegistry {
      */
     public async addTemplateToRegistry(templateUri: vscode.Uri, quiet?: boolean): Promise<void> {
         const pathAsString = pathutils.normalize(templateUri.fsPath)
+        this.assertAbsolute(pathAsString)
```

Comment:
Correct me if I'm wrong, but vscode.Uri should always be an absolute path? Not sure if this is necessary given that criteria.

(for instance, `vscode.Uri.file('asdf')` yields a `vscode.Uri` with `fspath: '/asdf'`), even on Windows)

Reply:
see https://github.com/aws/aws-toolkit-vscode/pull/1066#discussion_r420493925